WTWSMS/common/traits/WtWSMS_traits.txt

```diff
@@ -141,11 +141,14 @@ legendary_leader = {
 
 the_prophet = {
 	
-	muslim_opinion = 25
-	martial = 6
+	muslim_opinion = 100
+	martial = 12
 	learning = 12
  
 	ai_ambition = 1000
+	ai_zeal = 1000
+	ai_honor = -500
+	ai_rationality = 1
```

Comment:
One thousand ambition was already encoded before I came. One thousand zeal represents the zealous spread of Islam. Negative five hundred honor increases likelihood to break truces, dishonor treaties and alliances, and fight dirty through intrigue, all things that Muhammed totally mastered in his lifetime of conquest.